### ISSUE TITLE
Hi

### ISSUE THREAD

**Issue by rds99** [cesium-ml/cesium#272]
Is this library still in development or good for production level usage?

Thanks

**Comment by profjsb** [comment]
Yes, indeed! We're using it in a few projects at UC Berkeley. No warranties on its stability for real production usage, but it's been through fairly extensive testing over the years. Please do contribute!

**Comment by rds99** [comment]
Thanks, I saw error in front of build in readme.md section so thought will check, 
Is this similar to tsfresh?
